scanner/enricher/nvd/nvd.go

```diff
@@ -33,13 +34,10 @@ var (
 )
 
 const (
-	// Type is the type of data returned from the Enricher's Enrich method.
-	Type = `message/vnd.stackrox.scannerv4.vulnerability; enricher=nvd schema=https://csrc.nist.gov/schema/nvd/api/2.0/source_api_json_2.0.schema`
-
 	// DefaultFeeds is the default place to look for CVE feeds.
 	DefaultFeeds = `https://services.nvd.nist.gov/rest/json/cves/2.0/`
 
-	// This appears above and must be the same.
+	// This appears in [nvd.Type] and must be the same.
 	name = `nvd`
```

Comment:
Similar to the above with `fixedby`.

Reply:
I've tethered the name to the type in [04591be](https://github.com/stackrox/stackrox/pull/12310/commits/04591be133538e3ee3bb71d304d15619d495de5b), at the price of making the name public. Let's see if this raises concerns.